internal/codegen/blocks.go

```diff
@@ -46,20 +46,20 @@ func parseBlocks(filePath string) (map[string]string, error) {
 			case "Block":
 				blockName := matches[3]
 				if curBlockName != "" {
-					return nil, fmt.Errorf("invalid Block when already inside of a block, at %s:%d", filePath, i)
+					return nil, fmt.Errorf("invalid Block when already inside of a block, at %s:%d", filePath, i+1)
```

Comment:
Changes look good, but would be nice to add a test case for this scenario too.

Reply:
done